Pod/Classes/PINRemoteImageManager.m

```diff
@@ -192,11 +194,14 @@ - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)config
 
 - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)configuration alternativeRepresentationProvider:(id <PINRemoteImageManagerAlternateRepresentationProvider>)alternateRepProvider
 {
-    return [self initWithSessionConfiguration:configuration alternativeRepresentationProvider:nil imageCache:nil];
+    return [self initWithSessionConfiguration:configuration alternativeRepresentationProvider:nil imageCache:nil retryStrategyCreationBlock:nil];
```

Comment:
Is it necessary to provide this on init since there's a setter as well? It's possible that you could run requests before your setter executed, but super unlikely. I'm thinking it's not worth adding another parameter here?